pkg/apis/core/validation/validation_declarative.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
```

Comment:
Oh, I'd failed to notice the package of this file last review pass.  Do we need a copy of this in each api group package?  I was expecting to write once and share

Reply:
Yeah this really doesn't belong here. It doesn't have anything to do with the validation itself.  Do we have a util package under registry somewhere?

Or maybe a validation package under registry?

Reply:
I've moved this to same "rest" package as ValidateDeclaratively for now:

staging/src/k8s.io/apiserver/pkg/registry/rest/validation_declarative.go
staging/src/k8s.io/apiserver/pkg/registry/rest/validation_declarative_test.go